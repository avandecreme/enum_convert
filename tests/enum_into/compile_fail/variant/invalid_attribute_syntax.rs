use enum_convert::EnumInto;

#[derive(EnumInto)]
#[enum_into(Source)]
enum Source {
    #[enum_into(Target::Unit::Extra)] // Invalid syntax - too many segments
    Unit,
}

enum Target {
    Unit,
}

fn main() {}
