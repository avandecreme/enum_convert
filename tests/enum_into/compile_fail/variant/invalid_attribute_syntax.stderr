error: Expected Enum or Enum::Variant
 --> tests/enum_into/compile_fail/variant/invalid_attribute_syntax.rs:6:17
  |
6 |     #[enum_into(Target::Unit::Extra)]  // Invalid syntax - too many segments
  |                 ^^^^^^^^^^^^^^^^^^^
